Parent/Parent/Profile/Settings/View/SettingsGroupView.swift

```diff
@@ -0,0 +1,74 @@
+//
+// This file is part of Canvas.
+// Copyright (C) 2025-present  Instructure, Inc.
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Affero General Public License as
+// published by the Free Software Foundation, either version 3 of the
+// License, or (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU Affero General Public License for more details.
+//
+// You should have received a copy of the GNU Affero General Public License
+// along with this program.  If not, see <https://www.gnu.org/licenses/>.
+//
+
+import SwiftUI
+import Core
+
+public struct SettingsGroupView: View {
+    @ObservedObject var viewModel: SettingsGroupViewModel
+
```

Comment:
Let's add `@Environment(\.dynamicTypeSize) private var dynamicTypeSize` to make the title resize itself when the text size is changed. Subview resize themselves because the scaledMetric env var triggers an update.